### DISCUSSION TITLE
Multiselect or enable/disable range?

### DISCUSSION THREAD

**Discussion by AlainGamache** [easepick/easepick#43] [Q&A] [answered]
Hi! First of all I'd like to say I love easepick. It's the datepicker I didn't know I needed in my life.

I implemented it in one of my applications where people usually select on day at a time, but sometimes need to select more than a day. I thus enabled range, but some people are confused by having to click twice on the same day (start and end).

I'd like to know if there's a way to select multiple dates like with litepicker multiselect or if there are plans to add this as an upcoming feature? If not, is there a way to enable/disable the range plugin on the fly? My picker is inline, so I thought about adding a radio control for users to enable or disable range and make it disabled by default.

**Comment by wakirin** [comment, marked as answer]
Hi,
you can use `PluginManager` to manage plugins on fly: https://jsfiddle.net/waki/6zy1bw4t/
This example uses `@easepick/bundle` package, if you use `@easepick/core` package don't forget to include `@easepick/range-plugin` package.

Multiselect plugin for `easepick` is currently in development: https://github.com/easepick/easepick/tree/mselect-plugin

**Reply by AlainGamache** [reply]
Thanks for your prompt reply. It worked perfectly! I will make sure to try the multiselect plugin once it rolls out! :)
